platform/libs.junit4/nbproject/org-netbeans-libs-junit4.sig

```diff
@@ -617,9 +617,377 @@ meth public void testStarted(java.lang.String)
 supr junit.runner.BaseTestRunner
 hfds fPrinter
 
+CLSS public abstract org.hamcrest.BaseDescription
```

Comment:
Result of @sdedic opening up the module, so it is possible to use hamcrest checks in `NbModuleTest` I assume.